src/components/Patient/PatientFilter.tsx

```
@@ -782,3 +792,266 @@ export default function PatientFilter(props: any) {
+          paramKey: "last_consultation_is_telemedicine",
+        },
+        value(
+          "Discharge Reason",
+          "last_consultation__new_discharge_reason",
+          parseOptionId(
+            DISCHARGE_REASONS,
+            qParams.last_consultation__new_discharge_reason,
+          ) || "",
+        ),
+      ]}
+      children={
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)